mechanicalsoup/form.py

```diff
@@ -277,6 +277,24 @@ def set(self, name, value, force=False):
             return
         raise LinkNotFoundError("No valid element named " + name)
 
+    def __getitem__(self, name):
+        """Forwards arguments to :func:`~Form.get` to get value for a given form field
```

Comment:
We usually use an imperative tone for docstrings ("dear computer, please forward ..."), hence no 's'.